regex-syntax/src/hir/interval.rs

```diff
@@ -309,6 +309,11 @@ impl<I: Interval> IntervalSet<I> {
         }
         true
     }
+
+    /// Returns true iff this class is empty.
```

Comment:
Typo: `if`.

Reply:
I don't think it's a typo. `iff` is parlance for "if and only if."